plugins/CoreUpdater/templates/updateSuccess.twig

```diff
@@ -36,6 +36,17 @@
             </div>
         </div>
 
+        <h2>{{ 'CoreUpdater_UpdateLog'|translate }}</h2>
```

Comment:
now showing a log of messages... this can help troubleshoot issues and would show a message if for example plugins were disabled during the update etc.